pennylane/templates/layers/simplified_two_design.py

```diff
@@ -0,0 +1,192 @@
+from pennylane.ops import CZ, RY
+from pennylane.templates import broadcast
+from pennylane.templates.utils import (
+    _check_shape,
+    _check_wires,
+    _check_number_of_layers,
+    _check_type,
+    _get_shape,
+)
+
+
+@template
+def entangler(par1, par2, wires):
+    """Implements a two qubit unitary consisting of a controlled-Z entangler and Pauli-Y rotations.
+
+    Args:
+         par1 (float or qml.Variable): parameter of first Pauli-Y rotation
+         par2 (float or qml.Variable): parameter of second Pauli-Y rotation
+         wires (list): two wire indices that unitary acts on
+    """
+
+    CZ(wires=wires)
+    RY(par1, wires=wires[0])
+    RY(par2, wires=wires[1])
+
+
+@template
+def SimplifiedTwoDesign(initial_layer, weights, wires):
+    r"""
+    Layers consisting of a simplified 2-design architecture of Pauli-Y rotations and controlled-Z entanglers
+    proposed in `Cerezo et al. (2020) <https://arxiv.org/abs/2001.00550>`_.
+
+    A 2-design is an ensemble of unitaries whose statistical properties are the same as sampling random unitaries
+    with respect to the Haar measure up to the first 2 moments.
+
+    The template is not a strict 2-design, since
+    it does not consist of universal 2-qubit gates as building blocks, but has been shown in
+    `Cerezo et al. (2020) <https://arxiv.org/abs/2001.00550>`_ to exhibit important properties to study "barren plateaus"
+    in quantum optimization landscapes.
+
+    The template starts with an initial layer of single qubit Pauli-Y rotations, before the main
+    :math:`L` layers are applied. The basic building block of the main layers are controlled-Z entanglers
+    followed by a pair of Pauli-Y gates.
+    Each layer consists of an "even" part whose entanglers start with the first qubit,
+    and an "odd" part that starts with the second qubit.
+
+    This is an example of two layers, including the initial block:
+
+    .. figure:: ../../_static/templates/layers/simplified_two_design.png
+        :align: center
+        :width: 40%
+        :target: javascript:void(0);
+
+    The argument ``initial_layer`` contains the rotation angles of the initial layer of Pauli-Y rotations,
+    while ``weights`` contains the pairs of Pauli-Y rotation angles of the respective layers. Each layer takes
+    :math:`d = \left( \lfloor M/2 \rfloor + \lfloor (M-1)/2 \rfloor  \right)`` pairs of angles,
+    where :math:`M` is the number of wires.
+    The number of layers :math:`L` is derived from the first dimension of ``weights``.
+
+    Args:
+        initial_layer (array[float]): array of weights for the initial rotation block, shape ``(M,)``
+        weights (array[float]): array of rotation angles for the layers, shape ``(L, d, 2)``
+        wires (Sequence[int] or int): qubit indices that the template acts on
+
+    Raises:
+        ValueError: if inputs do not have the correct format
+
+    .. UsageDetails::
+
+        The template is used inside a qnode:
```

Comment:
Linking
```suggestion
        The template with two layers is used inside a :class:`~.BaseQNode`:
```

Reply:
Could we do this one:

```suggestion
        The template - here shown for two layers - is used inside a :class:`~.BaseQNode`::
```

Reply:
As per the other comment (I think on #555), we should link to the `qml.QNode` constructor, or `qml.qnode` decorator, here.

Reply:
Then:

```suggestion
        template - here shown for two layers - is used inside a :class:`~.QNode`:
```

Reply:
Looks good to me!